### PR TITLE
improving guide on custom tile loading

### PR DESCRIPTION
**Pull request by tsuz**
Fixes https://github.com/gis-ops/docker-valhalla/issues/17

### CODE REVIEW

**Comment by tsuz** [comment]
@nilsnolde May I ask for your review? I am not able to add people as reviewers...

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ If you need to customize Valhalla's configuration to e.g. increase the allowed m
 
 #### Run Valhalla with pre-built tiles
 
-In the case where you have a pre-built `tiles.tar` package from another Valhalla instance, you can also dump that to `custom_files/` and they're loaded upon container restart if you set the following environment variables: `use_tiles_ignore_pbf=True`, `force_rebuild=False`.
+In the case where you have a pre-built `tiles.tar` package from another Valhalla instance, you can also dump that to `custom_files/` and they're loaded upon container restart if you set the following environment variables: `use_tiles_ignore_pbf=True`, `force_rebuild=False`. Also, don't forget to set the correct `.file_hashes.txt` for your `valhalla_tiles.tar`.
```
Comment (nilsnolde) [inline README.md:72]: Suggested change:

Also, don't forget to set the md5 sum for your `valhalla_tiles.tar` in `.file_hashes.txt`.

**Review by nilsnolde — COMMENTED** [review]
Thanks, it's good to include that. Though again: that happens for you with `use_tiles_ignore_pbf=True`?

File: README.md (hunk 1 above)
Comment (tsuz) [reply README.md]: @nilsnolde Thanks. updated!

**Comment by tsuz** [comment]
@nilsnolde  The `use_tiles_ignore_pbf` has always been `True`...

**Review by nilsnolde — APPROVED** [review]
Thanks @tsuz

**Pull request merged**